x-pack/test/security_solution_cypress/cypress/e2e/exceptions/shared_exception_lists_management/shared_exception_list_page/read_only.cy.ts

```diff
@@ -35,8 +35,8 @@ describe('Shared exception lists - read only', { tags: ['@ess', '@skipInServerle
     // Create exception list not used by any rules
     createExceptionList(getExceptionList(), getExceptionList().list_id);
 
-    login(ROLES.reader);
-    visit(EXCEPTIONS_URL, { role: ROLES.reader });
+    login(ROLES.t1_analyst);
+    visit(EXCEPTIONS_URL, { role: ROLES.t1_analyst });
```

Comment:
Not needed to do it in this PR but something that would be nice to take into consideration for future iterations.

Looks like in the login method through the `cy.session` method we are caching the session we use when performing the login, so there is no need to explicitly pass the rule when visiting a page since should use the stored session. If so, we could simplify the code :)